### PR TITLE
[ADD] af_template_module: #221 Upgraded code quality for af_template_…

### PR DESCRIPTION
**Pull request by rupareliyahemangi145**
…module.

### CODE REVIEW

File: af_template_module/views/res_partner.xml [hunk 1]
```diff
@@ -1,80 +1,84 @@
...
-        <field name="inherit_id" ref="base.view_partner_form"/> 
-        <field name="arch" type="xml"> 
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
             <div name="button_box" position="inside">
                 <button name="hello_world" type="object">Hello world button</button>
             </div>
-            <page name="internal_notes" position="after"> <!-- position can be before, after or inside (if applied to a notebook, page or similar)-->
+            <page name="internal_notes"
```
Comment (MickeHolm) [inline af_template_module/views/res_partner.xml]: Keep old line and move the comment above this line

File: af_template_module/views/res_partner.xml [hunk 2]
```diff
@@ -1,80 +1,84 @@
...
+                  position="after"> <!-- position can be before, after or inside (if applied to a notebook, page or similar)-->
                 <page name="a_page" string="A test page"/>
             </page>
-            <xpath expr="//field[@name='phone']/.." position="after"> <!--more specific xpath in case greater specificity is necessary, in this case <field name=phone position="after"> caused everything to end up inside the field for some reason -->
+            <xpath expr="//field[@name='phone']/.."
```
Comment (MickeHolm) [inline af_template_module/views/res_partner.xml]: Keep old line and move the comment above this line

File: af_template_module/views/res_partner.xml [hunk 3]
```diff
@@ -1,80 +1,84 @@
...
+                   position="after"> <!--more specific xpath in case greater specificity is necessary, in this case <field name=phone position="after"> caused everything to end up inside the field for some reason -->
                 <label for="test_char_field" string="This is a custom label"/>
                 <field name="test_char_field" nolabel="1"/>
                 <field name="test_float_field"/>
                 <field name="test_integer_field"/>
                 <field name="test_datetime_field"/>
                 <field name="test_selection_field"/>
-                <field name="test_model_ids" widget="many2many_tags"/> <!--widgets can change the way a field is presented-->
-                <field name="test_related_field"/> 
+                <field name="test_model_ids"
```
Comment (MickeHolm) [inline af_template_module/views/res_partner.xml]: Keep old line and move the comment above this line

File: af_template_module/views/res_partner.xml [hunk 4]
```diff
@@ -1,80 +1,84 @@
...
             <tree string="Partners test">
-                <field name="test_related_field"/> 
+                <field name="test_related_field"/>
                 <field name="test_char_field"/>
                 <field name="test_float_field"/>
                 <field name="test_integer_field"/>
                 <field name="test_datetime_field"/>
                 <field name="test_selection_field"/>
-                <field name="test_model_ids" widget="many2many_tags"/> <!--widgets can change the way a field is presented-->
+                <field name="test_model_ids"
```
Comment (MickeHolm) [inline af_template_module/views/res_partner.xml]: Keep old line and move the comment above this line

File: af_template_module/views/res_partner.xml [hunk 5]
```diff
@@ -113,40 +117,45 @@
...
-        <field name="view_id" ref="view_partner_test_kanban" />
+        <field name="view_id" ref="view_partner_test_kanban"/>
     </record>
     <!--actions for each view type, to add other types just add the necessary view and another action like these for that type of view-->
-    <record id="action_view_partner_test_tree" model="ir.actions.act_window.view"> 
-        <field name="act_window_id" ref="action_partner_test" /> <!--the main action that can call this action-->
-        <field name="sequence" eval="2" /> <!-- determines the priority of the views, lowest number is loaded by default from the main action-->
+    <record id="action_view_partner_test_tree" model="ir.actions.act_window.view">
+        <field name="act_window_id" ref="action_partner_test"/> <!--the main action that can call this action-->
+        <field name="sequence"
```
Comment (MickeHolm) [inline af_template_module/views/res_partner.xml]: Keep old line and move the comment above this line

**Review by MickeHolm — CHANGES_REQUESTED** [review]

**Review by timforsstromAF — APPROVED** [review]

**Pull request closed without merging**